gradle-plugin/build.gradle

```diff
@@ -81,6 +82,16 @@ nexusPublishing {
     }
 }
 
+Boolean isReleaseVersion = !version.toString().endsWith("SNAPSHOT")
+
+signing {
+    setRequired({
+        isReleaseVersion && gradle.taskGraph.hasTask("publish")
+    })
+    useGpgCmd()
```

Comment:
Gradle will use JVM signing, `useGpgCmd()` is set because people such as myself have their signing key stored on a key device (i.e. YubiKey) which doesn't work with JVM signing.

Its also consistent with sbt-gpg (which also uses gpg).